packages/@aws-cdk/aws-backup/README.md

```diff
@@ -218,3 +223,74 @@ const role = new iam.Role(this, 'Access Role', { assumedBy: new iam.ServicePrinc
 
 importedVault.grant(role, 'backup:StartBackupJob');
 ```
+
+## Backup selection IAM role policies
+
+When creating a new `BackupSelection`, if no existing IAM role is provided, a
+new IAM role with a managed policy for backup will be created for the selection. 
+
+Many of the [AWS managed policies for AWS
+Backup](https://docs.aws.amazon.com/aws-backup/latest/devguide/security-iam-awsmanpol.html)
+ can be added to the auto generated or supplied IAM role using the flags:
+`allowBackup`, `allowRestores`, `allowS3Backup` and `allowS3Backups`. 
+
+The relevant service policies for each flag are shown in the table below: 
+
+| Field name        | AWS managed policy name                 | Default |
+|-------------------|-----------------------------------------|---------|
+| `allowBackup`     | AWSBackupServiceRolePolicyForBackup     | `true`  |
+| `allowRestores`   | AWSBackupServiceRolePolicyForRestores   | `false` |
+| `allowS3Backup`   | AWSBackupServiceRolePolicyForS3Backup   | `false` |
+| `allowS3Restores` | AWSBackupServiceRolePolicyForS3Restores | `false` |
+
+
+If you are creating a selection which includes an **S3 Bucket**, then the
+default service roles are not sufficient for backup nor restore operations. To
+enable S3 backups for a selection, include the `allowS3Backup` flag (the
+`AWSBackupServiceRolePolicyForS3Backup` managed policy). To enable S3 restores
+for a selection, include the `allowS3Restores` flag (the
+`AWSBackupServiceRolePolicyForS3Restore` managed policy). Both of these flags
+are `false` by default.
+
+For example, this selection would enable backup and restore permissions for an
+S3 bucket along with the standard backup and restore policies:
+
+```ts
+// Backup plan 
+declare const plan: backup.BackupPlan;
+
+// Bucket
+declare const bucket: s3.Bucket;
+
+// Selection
+const selection = new BackupSelection(stack, 'Selection', {
+  backupPlan: plan,
+  resources: [
+    BackupResource.fromS3Bucket(bucket),
+    // other resources here
+  ],
+
+  // Standard AWS Backup policies
+  allowRestores: true,
+
+  // S3 specific policies
+  allowS3Backup: true,
+  allowS3Restores: true,
```

Comment:
Question: would someone ever send in a s3 bucket in `resources` and not want to set `allowS3Backup`? To me, it seems like they go together and if so, we should find some way to determine if a resource is an s3 bucket and automatically add the s3 specific policy.

Same goes for `allowS3Restores`, I feel like we should be able to bake it into `allowRestores` and provide a better user experience.